archspec/cli.py

```diff
@@ -18,7 +19,36 @@ def main():
     """archspec command line interface"""
 
 
+def cb_cpu_dag(ctx, param, value):
```

Comment:
We could use a name with a leading underscore to hint that this function is not meant to be exported:
```suggestion
def _cpu_dag(ctx, param, value):
```